guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -1,7 +1,7 @@
 [id="installing-postgresql_{context}"]
 = Installing PostgreSQL
 
-You can install only the same version of PostgreSQL that is installed with the `{foreman-installer}` tool during an internal database installation. You can install PostgreSQL using Red Hat Enterprise Linux Server 7 repositories or from an external source, as long as the version is supported. {Project} supports PostgreSQL version 9.2.
+You can install only the same version of PostgreSQL that is installed with the `{foreman-installer}` tool during an internal database installation. You can install PostgreSQL using Red Hat Enterprise Linux Server 7 repositories or from an external source, as long as the version is supported. {Project} supports PostgreSQL version 12.1.
```

Comment:
Upstream still states PG 10+. On Ubuntu 18.04 this is the standard.

Also wondering if Red Hat Enterprise Linux Server 7 shouldn't use some macro so upstream can mention CentOS. 

However, the whole document is written very much with Satellite downstream in mind. Perhaps there should be a separate effort to make this guide really usable in upstream.

Reply:
I would like to keep it as a separate effort. This will require a review and tests of community members.